src/pages/Committee.tsx

```diff
@@ -67,19 +66,38 @@ interface State {
   committeeFref: firebase.database.Reference;
 }
 
+const CommitteeMedia = createMedia({
+  breakpoints: {
+    mobile: 320,
+    tablet: 768,
+    computer: 992,
+    largeScreen: 1200,
+    widescreen: 1920,
+  },
+});
+
+const mediaStyles = CommitteeMedia.createMediaStyle();
+const { Media, MediaContextProvider } = CommitteeMedia;
+
 
 class DesktopContainer extends React.Component<DesktopContainerProps, DesktopContainerState> {
   render() {
     const { body, menu } = this.props;
-
     // Semantic-UI-React/src/addons/Responsive/Responsive.js
     return (
-      <Responsive {...{ minWidth: Responsive.onlyMobile.maxWidth as number + 1 }}>
-        <Menu fluid size="small">
-          {menu}
-        </Menu>
-        {body}
-      </Responsive>
+      <>
+      <style>{mediaStyles}</style>
+      <MediaContextProvider>
```

Comment:
Something like this gets rid of some of the weird padding and borders (although there's still `1em` of padding even on a `<Segment basic />` 

```typescript
      <>
        <style>{mediaStyles}</style>
        <MediaContextProvider>
          <Segment as={Media} basic greaterThanOrEqual="tablet">
            <Menu fluid size="small">
              {menu}
            </Menu>
            {body}
          </Segment>
        </MediaContextProvider>
      </>
    );
```